src/backend/flags.rs

```diff
@@ -0,0 +1,62 @@
+use std::ops::{BitOr, BitOrAssign};
```

Comment:
I can add tests to this file if necessary.

Reply:
I added them so that the CI coverage tests would pass but it still decreased by 0.6%. I'm not really sure why, maybe it's counting clone and default implementations or something. It seems like it's ignoring integration_tests.rs for some reason, so maybe the new apply_filter* functions aren't getting covered.

I guess I'll try to figure out how to run them locally and either see if I can add more tests or just update the file.